expfmt/openmetrics_create.go

```diff
@@ -35,6 +35,12 @@ import (
 // sanity checks. If the input contains duplicate metrics or invalid metric or
 // label names, the conversion will result in invalid text format output.
 //
+// If metric names conform to the legacy validation pattern, they will be placed
+// outside the brackets in the traditional way, like `foo{}`. If the metric name
+// fails the legacy validation check, it will be placed quoted inside the
+// brackets: `{"foo"}`. As stated above, the input is assumed to be santized and
+// no error will be thrown in this case.
```

Comment:
Very good. But we also need a similar sentence about _label_ names (which are quoted if they contain formerly invalid characters).

Reply:
done (pending push)